README.md

```diff
@@ -57,6 +57,9 @@ Per-VDOM:
  * `fortigate_ipsec_tunnel_receive_bytes_total`
  * `fortigate_ipsec_tunnel_transmit_bytes_total`
  * `fortigate_ipsec_tunnel_up`
+ * `fortigate_wifi_access_points`
+ * `fortigate_wifi_access_point_client_count`
```

Comment:
The all-knowing metrics guide at prometheus.io says:
> The _sum, _count, _bucket and _total suffixes are used by Summaries, Histograms and Counters. Unless you’re producing one of those, avoid these suffixes.

I am also reading `https://xxx/api/v2/monitor/wifi/` and it says `Number of clients.` and `Retrieve statistics for all managed FortiAPs.`. I am a bit worried that it will be confusing with these vs. the `fortigate_wifi_ap_` family metrics so what about simply `fortigate_wifi_overall_clients` and `fortigate_wifi_overall_allowed_clients` ?

Reply:
I haven't been aware that this is the stat for all connected APs, but my impressions has been that this is the value for the specific AP we are querying.     

What do you think about `fortigate_wifi_mesh_clients` and `fortigate_wifi_mesh_allowed_max_clients`?

Reply:
Mesh has a very specific meaning in wifi, is it really mesh? Or could we use `_wifi_fabric_` maybe?

Reply:
Works for me!